docs/contributing/translation/sync-guide.md

```diff
@@ -4,6 +4,12 @@ title: Keeping Translations Up-to-date
 
 Periodically, gatsbybot will update your translation repo to be up-to-date with the current English repo. If there is an update to a page that is already translated, gatsbybot will create a pull request listing the conflicts between translation and the new English content. Resolving these conflicts and merging these pull requests is essential to keeping your translation repo up-to-date.
 
+## Gatsbybot sync behavior
+
+The gatsbybot will run the sync script on every translation repo on Friday at 6:00 PM PST. It will create up to two pull requests: one for changes that do not have conflicts, which can me immediately merged, and one for changes that do have conflicts, which will require updates and code review.
```

Comment:
```suggestion
```

Honestly, I think you may be able to get rid of this since it's explained in the next section. (provided we get more specific in the previous section) Maybe pull the previous paragraph into this section instead?